python/kserve/kserve/model_server.py

```diff
@@ -93,7 +98,8 @@ def __init__(self, http_port: int = args.http_port,
                  enable_latency_logging: bool = args.enable_latency_logging,
                  configure_logging: bool = args.configure_logging,
                  log_config: Optional[Union[Dict, str]] = args.log_config_file,
-                 access_log_format: str = args.access_log_format):
+                 access_log_format: str = args.access_log_format,
+                 http_middleware: Optional[typing.Type[BaseHTTPMiddleware]] = None):
```

Comment:
is it possible to support multiple middlewares in fastapi? as I check starlette has [example](https://www.starlette.io/middleware/) to support a list of middlewares.

Reply:
Yes that should be possible. Can add that to the PR if it is good with everyone @yuzisun @rachitchauhan43 @lizzzcai

Reply:
@lizzzcai / @Nagarajj :  Yeah, looks like good idea to me. We can start accepting list of middlewares instead of just single middleware.

Reply:
A chain is useful since we can aggregate multiple functions to it, auth, logging etc.